### DISCUSSION TITLE
Need help with components that are being dynamically imported in my app

### DISCUSSION THREAD

**Discussion by pritamReddyA** [storybookjs/storybook#25982] [Help]
### Summary

Hi guys, I recently started working on front end and our project uses dynamic imports( I hope they are called dynamic imports, where you do not need to mention the whole path of the file, just the last few ending lines) When i try to add these components as stories, the imports are not getting detected as they are all dynamic. I need a little help with how to proceed with integrating dynamic imports with storybook.

**Comment by valentinpalkovic** [comment]
Can you provide a minimal Stackblitz reproduction? https://storybook.new

**Reply by pritamReddyA** [reply]
Hi thanks a lot for responding, I fixed it, closing the discussion
